fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchNumberTest.java

```diff
@@ -274,19 +294,77 @@ public void testSearchNumber_decimal_or() throws Exception {
         assertSearchReturnsComposition("subject:Basic.decimal", "1,le100");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ge99.99");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ge99");
-        
+
         assertSearchReturnsComposition("subject:Basic.decimal", "ne13,1");
         assertSearchReturnsComposition("subject:Basic.decimal", "ap99.99,1");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ne13");
         assertSearchReturnsComposition("subject:Basic.decimal", "1,ap99.99");
     }
-    
-//    @Test
-//    public void testSearchNumber_decimal_chained_missing() throws Exception {
-//        assertSearchReturnsComposition("subject:Basic.decimal:missing", "false");
-//        assertSearchDoesntReturnComposition("subject:Basic.decimal:missing", "true");
-//        
-//        assertSearchReturnsComposition("subject:Basic.missing-decimal:missing", "true");
-//        assertSearchDoesntReturnComposition("subject:Basic.missing-decimal:missing", "false");
-//    }
-}
+
+    /*
+     * <PRE>
```

Comment:
the all-caps tag name feels very old-school :-)

Reply:
I didn't even realize I did it.

Reply:
no shouting in the next commit :)